index.html

```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
+                <h2 class="project1h">Professional Art Printing Data</h2>
+                <p class="project1p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="project1tags">
+                    <li class="htmlbuttp">
+                        <p class="icontextp">html</p>
+                        <a></a>
+                    </li>
+                    <li class="bootstrapbuttp">
+                        <p class="icontextp">bootstrap</p>
+                        <a></a>
+                    </li>
+                    <li class="rubybuttp">
+                        <p class="icontextp">Ruby</p>
+                        <a></a>
+                    </li>
+                </ul>
+                <ul class="seeproject1button">
+                    <li class="seeproject1">
+                        <button type="button" class="seeprojecttext">See Project</button>
+                        <a></a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <section class="aboutme">
+        <div class="aboutme1">
+            <h2 class="aboutmeh">About me</h2>
+            <p class="aboutmep">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <ul>
+                <li class="resume">
+                    <button type="button" class="resumeh">Get My Resume</button>
+                    <a></a>
+                </li>
+            </ul>
+        </div>
+        <div class="divline2"></div>
+        <div class="language">
+            <div class="languagetitle">
+                <div class="languagepoint"></div>
+                <h2 class="languageh">Language</h2>
+            </div>
+            <ul class="languagetext">
+                <li class="languagejavascript">Javascript</li>
+                <li class="languageruby">Ruby</li>
+                <li class="languagehtml">Html</li>
+                <li class="languagecss">Css</li>
+            </ul>    
+        </div>
+        <div class="frameworks">
+            <div class="frameworkstitle">
+                <div class="frameworkspoint"></div>
+                <h2 class="frameworksh">Frameworks</h2>
+            </div>
+            <ul class="frameworkstext">
+                <li class="frameworksbootstrap">Bootstrap</li>
+                <li class="frameworksrubyonrails">Ruby on Rails</li>
+                <li class="frameworksRSPec">RSPec</li>
+                <li class="frameworkscapybara">CapyBara</li>
+                <li class="frameworksselenium">Selenium</li>
+            </ul>
+        </div>
+        <div class="skills">
+            <div class="skillstitle">
+                <div class="skillspoint"></div>
+                <h2 class="skillsh">Skills</h2>
```

Comment:
- [ ] Please note that the technology categories ("Languages", "Frameworks" and "Skills") should be wrapped in a `ul`, each of them in a `li` tag e.g
```
<ul>
<li>
<h4>language</h4>
<ul>
<li>....</li>
<li>....</li>
<li>....</li>
<li>....</li>
</ul>
</li>
<li>
<h4> framework</h4>
<ul>
<li>....</li>
<li>....</li>
<li>....</li>
<li>....</li>
</ul>
</li>
.
.
.
.
.
</ul>
```